SINGE_Example.m

```diff
@@ -0,0 +1,47 @@
+%% Simple example that runs SCINGE for two replicates of two hyperparameter settings
+clear all;
+close all;
+clc;
+if ~isdeployed
+    addpath(genpath('.'));
+end
+
+%% Import list of parameter combinations
+fid = fopen('SINGE_params.cfg');
+temp = fgetl(fid);
+while any(temp~=-1)||isempty(temp)
```

Comment:
How much error handling do we need here?  I believe that `parseParams.m` will provide default values for most of the parameters.  Are the data files the only required configuration file values?